### DISCUSSION TITLE
Easily remove link parameters

### DISCUSSION THREAD

**Discussion by JohnDoe-JohnDoe** [U-C-S/Hurl#124] [Ideas]
Would it be possible to make it easier to remove the unnecessary parameters from a link?
Many links have additional parameters, usually for tracking, that are not needed. This is often everything after the "?". I realise the link can be manually edited (a good feature) but I would like it to be easier. On Android I use an app called URLcheck which does this with a single click, on the screenshot you see you can click "Apply"to remove the parameters thought to be useless or click the "x" to remove all or individual parameters.
![Screenshot_2024-06-09-17-22-14-534](https://github.com/U-C-S/Hurl/assets/66528253/42e980fa-c636-4524-8cbd-1cb1f81b92f9)

Could something like this be added to Hurl?

**Comment by U-C-S** [comment]
Auto detecting tracking params is something beyond scope of this project and neither there is a reasonable way to do it as far as I know.

>  see you can click "Apply"to remove the parameters thought to be useless or click the "x" to remove all or individual parameters.

This can be possible, by extending the current UI of Edit URL window. But will consider about this in the future.

**Reply by JohnDoe-JohnDoe** [reply]
Thanks for your reply.
The auto detect feature doesn't actually work that well, probably as you say it's hard to detect. I normally use the "x" which I have circled which removes everything after the "?". This nearly always works fine.
If you do manage to add this it would be much appreciated.
